pkgs/by-name/_1/_1password/package.nix

```diff
@@ -12,12 +23,14 @@ let
     if extension == "zip" then fetchzip args else fetchurl args;
 
   pname = "1password-cli";
```

Comment:
I see that there are `_1password` and `_1password-gui` in Nixpkgs currently.

What do you think about renaming this package from `_1password` to `_1password-cli`? Upstream calls this "1Password CLI" while the desktop application is just called `1Password`.

This is not a merge blocking review comment.

Reply:
Would probably be less confusing. I agree its confusing to have the cli own the root name.

Reply:
I'll create a follow up PR for that so we can handle that separately.